src/index.js

```diff
@@ -24,9 +24,9 @@ const server = http.createServer((req, res) => {
 
         if(Object.keys(notification).includes('image')) {
           const image = new Buffer(notification.image, 'base64');
-          await bot.sendPhoto(chatId, image, { caption: notification.message });
+          await bot.sendPhoto(chatId, image, { caption: notification.text });
```

Comment:
`notification.text` can be `undefined`, please add a check.  For example, if you use old version of hastic-server